types/material-ui/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for material-ui v0.16.0
+// Type definitions for material-ui v0.16.1
```

Comment:
@oliviertassinari or @mbrookes - you might have a clearer vision on what the types version should be in relation to the library version. I'll update it to whatever people want.